src/data/emacs-mode/agda-input.el

```diff
@@ -299,7 +299,7 @@ order for the change to take effect."
   (".-"        . ("∸"))
   (":"         . ,(agda-input-to-string-list "∶⦂ː꞉˸፥፦：﹕︓"))
   (","         . ,(agda-input-to-string-list "ʻ،⸲⸴⹁⹉、︐︑﹐﹑，､"))
-  (";"         . ,(agda-input-to-string-list "؛⁏፤꛶；︔﹔⍮⸵;"))
+  (";"         . ,(agda-input-to-string-list "⨟؛⁏፤꛶；︔﹔⍮⸵;"))
```

Comment:
If you insert at the beginning, aren't the other alternatives all shifted (and you have to press e.g.`\z;6` now instead `\z;5` previously)?  Wouldn't it in general make more sense to append at the end?  Or is it so in sync with colon?
(Note: I have not tested this PR.)

Reply:
@gallais : As the [author](https://github.com/agda/agda/commit/4d7076a89665335428d3952beb20a36bbacec4c3) of these lists, could you explain their ordering (and maybe suggest the best way forward here)?

Reply:
The ordering is probably inherited from whichever website had a list of all the unicode semicolons when I added them to the mode. I have no strong opinion on what to do here.

Reply:
I also found that there are (at least) two identical looking fat-white-semicolons:
- Z NOTATION SCHEMA COMPOSITION: https://www.fileformat.info/info/unicode/char/2a1f/index.htm 
- Z NOTATION RELATIONAL COMPOSITION: https://www.fileformat.info/info/unicode/char/2a3e/index.htm

Sigh.  Should we add both then?

Regarding position, there seems no canonical choice, so maybe in the same position (=2) as the fat-white-colon?

Reply:
They look different in my font, and in fact I like the "relational" one better (second below):
![semicolons](https://user-images.githubusercontent.com/646600/188280832-57775074-4287-40c5-8b7f-ff908fc45f99.png)
I would be in favor of having both.  As I said, I don't care about positioning.

Reply:
```suggestion
  (";"         . ,(agda-input-to-string-list "؛⨾⨟⁏፤꛶；︔﹔⍮⸵;"))
```